src/elm/Logs.elm

```diff
@@ -142,9 +174,11 @@ lineFocusButton stepNumber logFocus lineNumber clickAction shiftDown =
         [ Util.onClickPreventDefault <|
             clickAction <|
                 logRangeId stepNumber lineNumber logFocus shiftDown
-        , Util.testAttribute <| "log-line-num-" ++ String.fromInt lineNumber
+        , Util.testAttribute <| String.join "-" [ "log", "line", "num", stepNumber, String.fromInt lineNumber ]
         , id <| stepAndLineToFocusId stepNumber lineNumber
         , class "line-number"
+        , class "-link-ul"
+        , class "-ptr"
```

Comment:
making clickability of the log line numbers a bit more obvious 👍

Reply:
i feel like an existing `.button` class w/ modifier might work here. would `.button.-link` get us there? it has both underline and pointer.

Reply:
yeah it added some inadvertent style but I still like it :) 
<img width="455" alt="Screen Shot 2020-07-03 at 12 40 17 PM" src="https://user-images.githubusercontent.com/48764154/86490605-950c2300-bd2d-11ea-9c6a-880715faeef9.png">

Reply:
updated 👍